### PR TITLE
Home.md

### PR DESCRIPTION
**Pull request by sujitrs**
To add Introduction and explain the Network Architecture Diagram

### CODE REVIEW

**Comment by core-wg-admin** [comment]
@sujitrs can you please rebase and submit PR to `v0.5.1-draft` branch?

**Comment by core-wg-admin** [comment]
Layer 2 still says DHP.

File: Home.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+
+The details of the layers is as follows:
+
+## Layer I: Infrastructure Layer
+
+This layer consists of the underlying building blocks that are essential for the skill, employment and education ecosystem to come to life. These consists of functional registries of skill hubs, trainers, assessors, assessment agencies etc. as well as key components like Digital Identity, PKI Infrastructure to bring in Digital Signature capability, Payment and Settlement Infrastructure to enable digital payments, Data Protection and Consent Frameworks to ensure data privacy and consented data sharing, Document Store to store requisite document that needs to be exchanged, Rating and Reputation Infrastructure to provide facility to rate network participants and/or individuals, and online dispute resolution to address any grievances that arises out of transactions happening on DSEP Network.
+
+## Layer II: Interoperability Layer via. DHP Spec
```
Comment (core-wg-admin) [inline Home.md:22]: Needs to be corrected to DSEP

**Comment by sujitrs** [comment]
As requested, raised PR for branch - "beckn:v0.5.1-draft"

**Pull request closed without merging**